src/docs/asciidoc/Modules.asciidoc

```diff
@@ -264,6 +265,22 @@ In cases where you want to start e.g. a single-node runtime with a custom module
 `XD_CUSTOMMODULE_HOME=file\:/path/to/custom-modules bin/xd-singlenode`
 ====
 
+[[replicating-module-registry]]
+==== Replicating Module Registry
+When running in distributed mode, an alternative to using a shared file system for custom modules is to use the _replicating module registry_.
+
+If the value of `xd.customModule.home` does not use the `file:` protocol, then Spring XD will automatically set up a replicating registry that proxies that remote registry (at the end of the day, all modules need to be read from local filesystem anyway). This is all done transparently and by default, files are copied down from the central repository only if their contents has changed.
```

Comment:
We can remove "(at the end of the day, all modules need to be read from local filesystem anyway)".  This is covered below.